backend/Makefile

```diff
@@ -7,36 +7,38 @@ build:
 
 run:
 	-pkill docker-compose
-	DATABASE=postgres docker-compose up
+	docker-compose up postgres backend
 
 run_background:
 	-pkill docker-compose
-	DATABASE=postgres docker-compose up -d
+	docker-compose up -d postgres backend
 
 stop:
 	docker-compose down
 
 test:
 	docker build -t rafflebay/test .
 	-pkill docker-compose
-	DATABASE=postgres_test docker-compose up
+	docker-compose up --abort-on-container-exit postgres_test test
+	docker-compose down	
 
 format:
 	npx prettier --write --arrow-parens always --single-quote --trailing-comma all --no-bracket-spacing "src/**/*.js"
 	npx prettier --write --arrow-parens always --single-quote --trailing-comma all --no-bracket-spacing "test/**/*.js"
 
 # Open shell to the postgres instance
 p_shell:
-	DATABASE=postgres docker-compose exec postgres psql -U postgres rafflebay
+	docker-compose exec postgres psql -U postgres rafflebay
+
+test_p_shell:
+	docker-compose exec postgres_test psql -U postgres -p 5433 rafflebay_test
 
 #CAUTION: WILL DELETE ALL DATA
 reset_db: 
-	DATABASE=postgres docker-compose exec postgres psql -U postgres rafflebay -c "DROP TABLE payments, shipments, bids, items, users;"
+	docker-compose exec postgres psql -U postgres rafflebay -c "DROP TABLE bids, items, users, payments, shipments;"
```

Comment:
It actually needs to be in the order `DROP TABLE payments, shipments, bids, items, users` because of foreign key dependencies

Reply:
Fixed